package.json

```
@@ -41,6 +41,7 @@
     ]
   },
   "resolutions": {
+    "**/@graphql-tools/url-loader/**/ws": "7.4.6",
```

Maybe this, to continue receiving patch updates? :)

```suggestion
    "**/@graphql-tools/url-loader/**/ws": "^7.4.6",
```

---

Makes sense!

---

Done :)